### PR TITLE
Fixed #25680 -- Added django-admin shell --command option

### PR DESCRIPTION
**Pull request by nielsvanoch**
Add a -c option to the shell command to execute a command passed as a
string as Django.

### CODE REVIEW

File: docs/ref/django-admin.txt [hunk 1]
```diff
@@ -966,6 +966,10 @@ behavior you can use the ``--no-startup`` option. e.g.::
 
     django-admin shell --plain --no-startup
 
+You can also pass a command as a string to execute it as Django, like so::
+
+    django-admin shell --command="print 'test'"
```
Comment (raphaelm) [inline docs/ref/django-admin.txt]: Could you maybe think of an easy example that works on Python 2 and 3? :)

**Comment by raphaelm** [comment]
Apart from the comment above, this looks fine to me :)

File: docs/ref/django-admin.txt [hunk 2]
```diff
@@ -966,6 +966,10 @@ behavior you can use the ``--no-startup`` option. e.g.::
 
     django-admin shell --plain --no-startup
 
+You can also pass a command as a string to execute it as Django, like so::
+
```
Comment (hvdklauw) [inline docs/ref/django-admin.txt]: Should have a .. versionadded:: A.B

File: docs/ref/django-admin.txt [hunk 3]
```diff
@@ -966,6 +966,12 @@ behavior you can use the ``--no-startup`` option. e.g.::
 
     django-admin shell --plain --no-startup
 
+.. versionadded:: 1.10
```
Comment (timgraham) [inline docs/ref/django-admin.txt]: Please add a `.. django-admin-option::` heading and a mention in the 1.10 release notes.

File: tests/shell/tests.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+from django import __version__
+from django.core.management import call_command
+from django.test import SimpleTestCase
+from django.test.utils import patch_logger
+
+
+class ShellCommandTestCase(SimpleTestCase):
+
+    def test_command_option(self):
+        with patch_logger('test', 'info') as logger:
```
Comment (timgraham) [inline tests/shell/tests.py]: Is the complexity of testing with a logger needed? I think a print statement and passing `stdout=StringIO()` to `call_command()` in order to check the value would be sufficient.

**Comment by timgraham** [comment]
Please squash commits, use the pull request title as the commit message, and uncheck "Patch needs improvement" on the ticket. Thanks!

**Comment by timgraham** [comment]
buildbot, add to whitelist.

File: tests/shell/tests.py (hunk 4 above)
Comment (nielsvanoch) [inline tests/shell/tests.py]: I don't think that will work (or at least I couldn't get it to work): the print statement would be ```exec```ed, which would not redirect it to stdout as with normal management command prints. I left it the way it is, unless you can show me a better solution.

**Comment by timgraham** [comment]
merged in 7f7553dd30534d606c84952a3f6dcb64b396ce37, thanks!

**Pull request closed without merging**